src/app/exportAtsumaru.ts

```diff
@@ -32,6 +32,13 @@ export function promiseExportAtsumaru(param: ExportHTMLParameterObject): Promise
 			if (!gameJson.environment) {
 				gameJson.environment = {};
 			}
+			if (!gameJson.environment.niconico || !gameJson.environment.niconico.supportedModes) {
+				// モード指定がなければ、常に指定可能なモードであるsingleモードを追加する。
+				completedParam.logger.warn("Supported-modes must be specified, so \"single\" has been added");
```

Comment:
プロパティ名は厳密に表記したいです (ぐぐった時にあたりやすくしたいので)。説明としても「game.jsonの内容のことである」「--atsumaruの時限定の話である」とかも内容に込められると良さそうなので、例えばこんな感じでどうでしょう。

```suggestion
				completedParam.logger.warn("'environment.niconico.supportedModes', a required property  for '--atsumaru' mode, is not given in game.json. Assumed to be [\"single\"].");
```

(コミット内容を直接提案する新機能が Github についたみたいなので使ってみましたが、このままだと1行が長すぎてlintに怒られるかもしれません)

Reply:
こんな機能ができたんですね。。そちらの文言が良さげだったのでコミットしました。lintで怒られてたら修正しようと思います。